src/shared/components/Stepper/Stepper.tsx

```diff
@@ -1,47 +1,50 @@
 import React, { useEffect, useState } from 'react'
 
-import { SvgGlyphFileVideo, SvgGlyphLock, SvgGlyphTrash } from '@/shared/icons'
-import { FileType } from '@/types/files'
+import { SvgGlyphCheck, SvgGlyphFileVideo, SvgGlyphLock, SvgGlyphTrash } from '@/shared/icons'
 
 import {
-  FileName,
   Overhead,
   StepDetails,
+  StepImage,
   StepNumber,
   StepStatus,
+  StepTitle,
   StepWrapper,
   StyledProgress,
-  Thumbnail,
-} from './FileStep.style'
+} from './Stepper.styles'
 
 import { IconButton } from '../IconButton'
 
-export type FileStepProps = {
-  stepNumber: number
-  active: boolean
-  onDelete: (e: React.MouseEvent<HTMLButtonElement, MouseEvent>) => void
-  type: FileType
-  isFileSet?: boolean
+export type StepperProps = {
+  title: string
+  variant?: 'file' | 'default'
+  completed?: boolean
   thumbnailUrl?: string | null
-  onSelect?: (step: FileType) => void
-  disabled?: boolean
   isLoading?: boolean
+  type?: 'video' | 'image'
```

Comment:
How about we remove this prop and replace it with something generic like `icon: React.ReactNode`?